### PR TITLE
Add `jobs` command

### PR DESCRIPTION
**Pull request by k0va1**
Here i implement `/jobs` command. If you want to add new job you need to add a record to `data/jobs.yaml`. Every job should have: `id`, `title`, `company`, `location`, `announce`(short description of the job) and `full_description`. When you go to `/jobs` you'll get all jobs in separated messages. Every job message has a button(More) to get full description.

p.s. code is not very good, but it works. Any suggestions and comments are welcome :)

### CODE REVIEW

File: dispatcher.rb [hunk 1]
```diff
@@ -67,12 +80,13 @@ def speakers(ctx)
   def places(ctx)
   end
 
-  def job_board(ctx)
+  def jobs(ctx)
+    Dispatchers::Job.new(bot).jobs(ctx)
```
Comment (IvanShamatov) [inline dispatcher.rb:84]: you can memoize it

File: dispatcher.rb [hunk 2]
```diff
@@ -67,12 +80,13 @@ def speakers(ctx)
...
   end
 
   def vote(ctx)
     vote_button = Telegram::Bot::Types::InlineKeyboardMarkup.new(
-      inline_keyboard: [Telegram::Bot::Types::InlineKeyboardButton.new(text: 'Like', callback_data: 'like')]
+      inline_keyboard: [Telegram::Bot::Types::InlineKeyboardButton.new(text: 'Like', callback_data: { command: 'like' }.to_json)]
```
Comment (IvanShamatov) [inline dispatcher.rb:89]: ❤️

File: dispatchers/job.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    def more(callback)
+      data = JSON.parse(callback.data)
+      job = JOBS.detect { |j| j['id'] == data['job_id'] }
+      text = "*#{job['title']}*\nCompany: #{job['company']}\nLocation: #{job['location']}\n#{job['announce']}\n#{job['full_description']}"
+      bot.api.edit_message_text(
+        message_id: callback.message.message_id,
+        chat_id: callback.message.chat.id,
+        text: text,
+        parse_mode: :markdown
```
Comment (IvanShamatov) [inline dispatchers/job.rb:19]: will `more` button change to `less` button. It seems it's not rocket science to update keyboard too

File: dispatchers/job.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      )
+    end
+
+    def jobs(ctx)
+      JOBS.each_with_index do |job, i|
```
Comment (IvanShamatov) [inline dispatchers/job.rb:24]: We actually can split data/messages/extended messages preparation from transport API. WDYT?

File: dispatcher.rb [hunk 5]
```diff
@@ -30,10 +34,19 @@ def call(message)
...
+    rescue JSON::ParserError
+      return nil
+    end
+
+    if command == 'like'
       # redis.publish('liker_bot', message.from.username)
-      bot.api.answer_callback_query(callback_query_id: message.id)
+      # Dispatchers::Like.new(...)
+      bot.api.answer_callback_query(callback_query_id: callback.id)
+    elsif command == 'more'
```
Comment (IvanShamatov) [inline dispatcher.rb:48]: I suggest using "job_more" or something with `job` prefix here as it is global namespace of commands

File: dispatcher.rb (hunk 1 above)
Comment (k0va1) [reply dispatcher.rb:84]: good point

File: dispatchers/job.rb (hunk 3 above)
Comment (IvanShamatov) [reply dispatchers/job.rb:19]: what can we do here, is actually split `less` functionality into another task

**Pull request merged**